support/test_plan.rb

```diff
@@ -154,10 +154,6 @@ def run_test_with_libraries(*libraries)
       option_parser.parse!
     end
 
-    RSpec.configure do |config|
-      config.color_mode = color_enabled? ? :on : :off
-    end
```

Comment:
@mcmire have a look at this regarding the desire to test the bug fix. I think the reason the bug wasn't causing tests to fail in master is this line was copying the SuperDiff config to the RSpec config.

This was effectively hiding the fact that SuperDiff was inconsistently looking at both of these configs in two different places.

Removing this line makes the code in master fail specs, while the code in my PR here still passes all specs.

Reply:
Aha! Good find. Makes perfect sense!